src/components/Footer/Footer.test.tsx

```diff
@@ -0,0 +1,8 @@
+import { customRender } from 'custom-render';
```

Comment:
had to add a random jest jest, otherwise jest on the pipeline would fail. deleted all the other jest tests when deleting unused components